### PR TITLE
Add url to deployment page

### PR DESCRIPTION
**Pull request by BenMQ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # codeu-starter-project
 This is the shared repository for team 45 for the Summer '19 APAC CodeU cohort. 
 
+Our project is deloyed [here](http://su19-codeu-45-721.appspot.com/).
```
Comment (Jay-kew) [inline README.md]: Should we use the https version?

**Review by Jay-kew — APPROVED** [review]
Thanks for adding the url! Makes it easier to access rather than searching the chat everytime haha

**Review by calebmah — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (calebmah) [reply README.md]: +1 for the https version!

Comment (BenMQ) [reply README.md]: Didn't know it was configured, done!

**Pull request merged**